jtransc-rt/resources/cpp/Base.cpp

```diff
@@ -495,18 +523,45 @@ struct JA_I : JA_Base<int32_t> {
 	static JA_I *fromArgValues(int32_t a0, int32_t a1, int32_t a2) { return (JA_I * )(new JA_I(3))->init(0, a0)->init(1, a1)->init(2, a2); };
 	static JA_I *fromArgValues(int32_t a0, int32_t a1, int32_t a2, int32_t a3) { return (JA_I * )(new JA_I(4))->init(0, a0)->init(1, a1)->init(2, a2)->init(3, a3); };
 
+	static JA_I* fromArray(std::wstring desc, std::vector<int32_t> array) {
```

Comment:
@intrigus do you know how can I make generic all those methods with templates? so I can avoid creating duplicating them?